wire-ios-request-strategy/Sources/Request Strategies/Conversation/Actions/CreateConversationGuestLinkActionHandlerTests.swift

```diff
@@ -0,0 +1,155 @@
+//
+// Wire
+// Copyright (C) 2024 Wire Swiss GmbH
+//
+// This program is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// This program is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with this program. If not, see http://www.gnu.org/licenses/.
+//
+
+import WireDataModelSupport
+import XCTest
+
+@testable import WireRequestStrategy
+
+final class CreateConversationGuestLinkActionHandlerTests: ActionHandlerTestBase<CreateConversationGuestLinkAction, CreateConversationGuestLinkActionHandler> {
+
+    // MARK: - Properties
+
+    private var stack: CoreDataStack!
+    private let coreDataStackHelper = CoreDataStackHelper()
+    private var conversationID: UUID!
+
+    private var syncContext: NSManagedObjectContext {
+        return stack.syncContext
+    }
+
+    override func setUp() async throws {
+        try await super.setUp()
+        stack = try await coreDataStackHelper.createStack()
+
+        await syncContext.perform { [self] in
+
+            conversationID = UUID()
+
+            self.action = CreateConversationGuestLinkAction(password: nil, conversationID: conversationID)
+            self.handler = CreateConversationGuestLinkActionHandler(context: syncContext)
+        }
+    }
+
+    override func tearDown() async throws {
+        stack = nil
+        try coreDataStackHelper.cleanupDirectory()
+        try await super.tearDown()
+    }
```

Comment:
There is one memory problem with unit tests, that is not obvious: they are not deallocated and exist in memory until end of executing the whole test suite / testplan. https://qualitycoding.org/xctestcase-teardown/

Therefore all properties should be released (except some minor constants etc.), especially because it still can have side effects if they listen to notifications or similar.
Please release `coreDataStackHelper` and `conversationID` too :)

Reply:
Done!